parent/pom.xml

```diff
@@ -3030,6 +3030,7 @@
             <artifactId>flaky-test-extractor-maven-plugin</artifactId>
             <configuration>
               <reportDir>${project.build.directory}/${flaky.test.reportDir}</reportDir>
+              <failBuild>false</failBuild>
```

Comment:
This helps for the main goal of not failing the build on duplicate tests (that got retried due to flakiness)